### PR TITLE
update to support elixir 1.9+

### PR DESCRIPTION
**Pull request by erikosmond**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,17 +69,18 @@ If you have a variable in your system environment that you want injected in an a
 
 ## Installation
 
-The correct version for your needs depends on the version of Distillery your project uses:
+The correct version for your needs depends on the version of Distillery or Elixir your project uses:
 
 - For Distillery versions `2.1.0` and later, you want `{:aws_ssm_provider, "~> 1.0.0"}`.
 - For Distillery versions `2.0.14` and earlier, you want `{:aws_ssm_provider, "~> 0.2.1"}`.
+- For Elixir versions `1.9` and later (without Distillery), you want `{:aws_ssm_provider, "~> 2.0.0"}`.
```
Comment (shonfeder) [inline README.md:76]: Maybe put this on line 73, so the version are in descending order.

**Review by shonfeder — APPROVED** [review]
One suggestion, but LGTM.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 # aws_ssm_provider
 
-A configuration provider for Distillery which handles AWS SSM parameters.
+A configuration provider for Mix which handles AWS SSM parameters.
 
 ## Usage
 
-This package is intended to be used in conjunction with [Distillery Config Providers](https://hexdocs.pm/distillery/config/runtime.html#config-providers).
+This package is intended to be used in conjunction with Mix Config.Providers in Elixir 1.9+
```
Comment (dillonius01) [inline README.md:7]: Question: do we want to note that in `v1.0`, we supported Distillery, but as of `v2.0` we only support the native Mix releases?

**Review by dillonius01 — APPROVED** [review]
🚢

File: README.md (hunk 2 above)
Comment (erikosmond) [reply README.md:7]: @dillonius01 That's what line 76 in the README is for.

**Pull request merged**